src/site/content/en/metrics/lcp/index.md

```diff
@@ -241,54 +249,24 @@ po.observe({type: 'largest-contentful-paint', buffered: true});
 // leaves the tab.
 addEventListener('visibilitychange', function fn() {
   if (lcp && document.visibilityState === 'hidden') {
-    sendToAnalytics({'largest-contentful-paint': lcp});
+    console.log('LCP:', lcp);
```

Comment:
Is there a way for this API to detect the 'initial' state? It seems this would still report if the page is backgrounded from the start (listener won't trigger) - then foregrounded, bad LCP captured, then backgrounded.

Reply:
This won't trigger if the state changes from `hidden` to `visible`. So it should be fine with that case.